app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -94,7 +94,11 @@ def provided_attributes
 
     #FIXME
```

Comment:
Can we remove this FIXME? is 16 the max count supported for these by ovirt?

Reply:
@tbrisker 
After looking at ovirt engine,  I saw the following:
[root@compute-ge-6 ~]#  engine-config --get MaxNumOfVmCpus
MaxNumOfVmCpus: 240 version: 3.6
MaxNumOfVmCpus: 240 version: 4.0
MaxNumOfVmCpus: 288 version: 4.1
MaxNumOfVmCpus: 384 version: 4.2

[root@compute-ge-6 ~]# engine-config --get  MaxNumOfCpuPerSocket
MaxNumOfCpuPerSocket: 16 version: 3.6
MaxNumOfCpuPerSocket: 16 version: 4.0
MaxNumOfCpuPerSocket: 254 version: 4.1
MaxNumOfCpuPerSocket: 254 version: 4.2

that means the max_socket=16 and max_cores=16 is true to ovirt 3.6 and 4.0 but not to versions >= 4.0 
After taking to @orrabin , we think that at the moment we don't see any reason to set the max socket and cores to be more then 16.
Until now will allowed only 8 cores and after this patch, we will allow 16*16 = 256 cores, which seems to be enough at the moment.